### DISCUSSION TITLE
Formato Tests Parte 2

### DISCUSSION THREAD

**Discussion by maxo2727** [IIC2133-PUC/2024-1#151] [Tarea 2]
Hola. Al descargar los tests de la parte 2, me aparecen en un formato distinto, y no se pueden entender. Debo cambiar algo en mi block de notas para leer bien el test u otra cosa?

![image](https://github.com/IIC2133-PUC/2024-1/assets/127250553/662c56e6-7450-4c78-893f-f1b44a583415)

**Comment by benjavicente** [comment]
Hola! Necesito más información para poder replicarlo. ¿Qué teste específicamente? ¿Trataste de abrirlo en otra aplicación? ¿En drive, puedes ver ese test?
